snuba/datasets/configuration/storage_builder.py

```diff
@@ -81,13 +81,15 @@ def build_stream_loader(loader_config: dict[str, Any]) -> KafkaStreamLoader:
     ).from_kwargs()
     default_topic = Topic(loader_config["default_topic"])
     # optionals
-    pre_filter = (
-        CONF_TO_PREFILTER[loader_config[PRE_FILTER]["type"]](
+    pre_filter = None
+    if PRE_FILTER in loader_config and loader_config[PRE_FILTER] is not None:
```

Comment:
looks like you've added extra logic. do you have a test for that?

Reply:
This if statement logic is the same, but isn't written inline. Writing it this way save us a few extra lines of code.